apps/ios/GuideDogs/Code/Data/Models/JSON Parsing/OSM/GeoJsonGeometry.swift

```diff
@@ -318,7 +318,7 @@ public enum GeoJsonGeometry: Equatable, Codable {
 
 extension GeoJsonGeometry {
     private static func into_coord_pair(_ coord: CLLocationCoordinate2D) -> [CLLocationDegrees] {
-        return [coord.longitude, coord.latitude]
+        return [coord.latitude, coord.longitude]
```

Comment:
@RDMurray is this the latitude/longitude swap in JSON parsing that you caught? Which one was correct?